src/playbooks/with_build/vstat_security_hardening.yml

```diff
@@ -0,0 +1,7 @@
+---
+- hosts: vstats,data_vstats
```

Comment:
In vstat-deploy, the security hardening is executed for vstats, data_vstats, and primary vstats. Here primary vstats is not part of the hosts. Shouldn't the hosts be consistent?

Reply:
done